pwiz_tools/Skyline/Test/CodeInspectionTest.cs

```diff
@@ -215,6 +219,50 @@ List<string> CheckFormsWithoutTestRunnerLookups()
             return missing;
         }
 
+        /// <summary>
+        /// We have some code, especially in commandline, that expects certain error messages to start with "Error:" or the L10N equivalent
+        /// </summary>
+        void InspectConsistentErrorMessages(List<string> errors)
+        {
+            var currentCulture = Thread.CurrentThread.CurrentUICulture;
+            try
+            {
+                Thread.CurrentThread.CurrentCulture = Thread.CurrentThread.CurrentUICulture = new CultureInfo("en"); // We want to compare against the "en" resources
+                var resourceSet = Skyline.Properties.Resources.ResourceManager.GetResourceSet(Thread.CurrentThread.CurrentUICulture, true, true);
+
+                // Build the string of resources.
+                foreach (var resource in resourceSet)
+                {
+                    var pair = resource as DictionaryEntry? ?? default;
+                    var val = pair.Value as string ?? string.Empty;
+                    if (val.StartsWith("Error:"))
```

Comment:
I would try to make this always consistent with what is happening in CommandStatusWriter, so that they actually use the same string. Seems unlikely that the text will ever change, but this is a magic text value that is required to match in two places, and I just try to stay away from those as a rule.

Reply:
I would probably just use the "en" resourceSet CommandStatusWriter_WriteLine_Error_ for this value immediately after you make it the current language, and then programmatically refer to it below.

Reply:
Sounds good. There's a subtle difference between that resource and that string constant, one has a traling space and one does not. I'll make the string constant public and use it in both places.